pipelines/HSC/DRP-RC2_subset.yaml

```diff
@@ -0,0 +1,427 @@
+      skyCorr, it requires multiple detectors per visit. Although they duplicate
+      tasks from the main "stepN" subsets from drp_pipe's DRP-RC2.yaml, we
+      include the subsets singleFrame, source_calibration, coadd_measurement, and
+      forced_objects within this pipeline so that users following the "getting
+      started" tutorial on pipelines.lsst.io understand the basic steps that are
+      contained within each subset.
+  source_calibration:
+    subset:
+      - finalizeCharacterization
+      - updateVisitSummary
+      - makeCcdVisitTable
+      - makeVisitTable
+      - writeRecalibratedSourceTable
+      - transformSourceTable
+      - consolidateSourceTable
+  coadd_measurement:
+    subset:
+      - detection_tutorial
+      - mergeDetections
+      - deblend
+      - measure
+      - mergeMeasurements
+  forced_objects:
+    subset:
+      - forcedPhotCcd
+      - forcedPhotCoadd
+      - transformObjectTable
+      - writeObjectTable
+      - consolidateObjectTable
+  nightlyStep1:
+    subset:
+      - isr
+      - characterizeImage
+      - calibrate
+      - writePreSourceTable
+      - transformPreSourceTable
+    description: >
+      Per-detector tasks that can be run together to start the DRP pipeline.
+      These should never be run with 'tract' or 'patch' as part of the data ID
+      expression if any later steps will also be run, because downstream steps
+      require full visits and 'tract' and 'patch' constraints will always
+      select partial visits that overlap that region.
+  nightlyStep2a:
+    subset:
+      - isolatedStarAssociation
+      - consolidatePreSourceTable
+      - consolidateVisitSummary
+      - skyCorr
+    description: >
+      Per-visit tasks that can be run together, but only after the 'step1'.
+      These should never be run with 'tract' or 'patch' as part of the data ID
+      expression. skyCorr and FGCM require full visits and 'tract' and 'patch'
+      constraints will always select partial visits that overlap that region.
+      This includes FGCM because it's configured here to run in "global" mode,
+      which means one should not use 'tract' expression to constrain it, and if
+      one _did_ run it with a tract constraint (which would be a common
+      occurrence if it was included in any later step), it would be fed the
+      wrong (partial-visit) inputs to its 'background' connection.
+      This subset is considered a workaround for missing middleware and task
+      functionality.  It may be removed in the future.
+  nightlyStep2b:
+    subset:
+      - jointcal
+    description: >
+      Per-tract tasks.
+      jointcal should be run with explicit 'tract' constraints essentially all the
+      time, because otherwise quanta will be created for jobs with only partial
+      visit coverage.
+      isolatedStarAssociation can be run with or without 'tract' constraints.
+  nightlyStep2cde:
```

Comment:
Does it work to include FGCM in the same step with the other tasks? In rc2_subset, FGCM has been separate because it explicitly has to be run on a single processor. Is that true of the other tasks here, or would it improve performance to keep them separate?

Reply:
See the shell script that executes rc2_subset processing at https://github.com/lsst/rc2_subset/blob/bf14286d29ed909cd0b0949231dc2b19725cb8d1/bin/measureHscRC2Metrics.sh

Reply:
Thanks Jeff. Following a conversation with @erykoff, I've decided to revert this change and maintain the three FGCM tasks in their own `nightlyStep3c` subset, with the remainder continuing on as before in `nightlyStep2d`. To that end, the only actual pipeline changes should now be the addition of `makeVisitTable` and `makeCcdVisitTable` into `nightlyStep2d`. Other proposed changes, such as moving `isolatedStarAssociation` from `nightlyStep2b` into `nightlyStep2a` (as in `DRP-RC2.yaml` have also been dropped).